### PR TITLE
[script.plex] 0.2.0

### PR DESCRIPTION
**Pull request by ruuk**
### Description

Was previously submitting to Krypton, so [here's a link to the diff of changes](https://github.com/plexinc/plex-for-kodi/compare/b4ab4c95f07a608a3ea2bd05b1e3e12bd6122b1a...511bcc937efe3b40e53880b9e12d53eac34d5117).

The main changes here are for python 3 compatibility (there's otherwise one fix and 2 language additions). Once this is in I'll open a PR against matrix and supply a diff (which will just be the addon.xml version changes).

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by basrieter** [comment]
are there many changes compared to the 0.1.8 from Krypton (in https://github.com/xbmc/repo-scripts/pull/1368)

**Comment by ruuk** [comment]
> are there many changes compared to the 0.1.8 from Krypton (in #1368)

What's in the diff I linked to is the changes since the last push to Krypton. #1368 just has a fix cherry picked from the branch this PR is pulled from.

File: script.plex/addon.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        <summary lang="es">Plex para Kodi</summary>
+        <summary lang="ru">Plex for Kodi</summary>
+        <description lang="en">Official Plex for Kodi add-on</description>
+        <description lang="de">Offizielles Plex für Kodi Addon</description>
+        <description lang="hu">Hivatalos Plex for Kodi add-on</description>
+        <description lang="it">Addon ufficiale Plex per Kodi</description>
+        <description lang="es">Addon oficial Plex para Kodi</description>
+        <description lang="fr">Add-on Officiel Plex pour Kodi</description>
+        <description lang="ru">Официальный аддон Plex for Kodi</description>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```
Comment (enen92) [inline script.plex/addon.xml:33]: Spdx

File: script.plex/addon.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    </requires>
+    <extension point="xbmc.python.script" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.python.pluginsource" library="plugin.py">
+        <provides>executable</provides>
+    </extension>
+    <extension point="xbmc.service" library="service.py" />
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Plex for Kodi</summary>
```
Comment (enen92) [inline script.plex/addon.xml:20]: Please fix the language codes en_GB, de_DE, etc

**Pull request merged**